src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,41 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  const handleTabSelected = tabId => {
+    if (activeTabId !== tabId) {
+      onTabSelected(tabId);
+    }
+  };
+
+  const currentTab = tabs.find(tab => tab.id === activeTabId);
```

Comment:
```suggestion
  const currentTab = tabs.find(tab => tab.id === activeTabId) || tabs[0];
```
Just set the backup value so you don't need any other variables